src/drive/styles/actionmenu.styl

```diff
@@ -46,3 +46,10 @@
     background embedurl("../assets/icons/icon-rename.svg") 1rem center no-repeat
 .fil-action-share
     background embedurl("../assets/icons/icon-share-grey.svg") 1rem center no-repeat
+    position   relative
+    
+    > div
```

Comment:
I think `div` as a selector here is a bit confusing. Wouldn't we be better off giving it a class like `action-extra` or something like that?